core/templates/dev/head/pages/story_viewer/story_viewer_navbar_breadcrumb_directive.html

```diff
@@ -0,0 +1,6 @@
+<div class="nav navbar-nav oppia-navbar-breadcrumb ng-cloak">
+  <span class="oppia-navbar-breadcrumb-separator"></span>
+  <span>
+    <[storyTitle]>
```

Comment:
Should we have the topic name in the breadcrumb?

Reply:
Yes, this could be done. In the breadcrumb, we can have the topic title and we can display story title in the banner.